src/core/lib/run.js

```diff
@@ -9,7 +8,6 @@ module.exports = function(query, callback) {
   } else {
     queries.push(query);
   }
-  request = new Request(this, queries, cb).refresh();
-  cb = callback = null;
```

Comment:
This callback referencing technique helps avoid a memory leak that  can occur when functions are passed around this way.

More here: http://info.meteor.com/blog/an-interesting-kind-of-javascript-memory-leak